eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/AssemblyCheckerProcessor.java

```diff
@@ -57,7 +59,9 @@ private String getSequenceUsingSynonyms(ContigSynonyms contigSynonyms, long star
         if ((sequence = getSequence(contigSynonyms.getUcsc(), start, end)) != null) {
             return sequence;
         }
-        throw new IllegalArgumentException("Contig " + contigSynonyms.getSequenceName() + " not found in FASTA file");
+        throw new IllegalArgumentException("Contigs " + contigSynonyms.getSequenceName() + ", " + contigSynonyms
```

Comment:
well, I thought about doing a toString on contigSynonyms. like this it seems they are 4 unrelated contigs